boards/shields/x_nucleo_nfc04a1/Kconfig.defconfig

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2023 Kickmaker
+# SPDX-License-Identifier: Apache-2.0
+
+if SHIELD_X_NUCLEO_NFC04A1
+
+if EEPROM
+
+config EEPROM_ST25DV
+	default y
+
+endif # EEPROM
+
+endif # SHIELD_X_NUCLEO_NFC04A1
```

Comment:
This should not be needed. `CONFIG_EEPROM_ST25DV` should automatically be enabled based on the shield DTS.

Reply:
I removed the defconfig. It did not work before because I was missing the DT_HAS from https://github.com/zephyrproject-rtos/zephyr/pull/62599#discussion_r1333518581